src/app/routes/article/getInitialData/index.ts

```diff
@@ -51,33 +49,6 @@ export default async ({
       ...(!isLocal && { agent, optHeaders }),
     });
 
-    // Ensure all local CPS fixture and test data is in the correct format
-    if (isLocal && pageType === 'cpsAsset') {
-      const secondaryData = await getAdditionalPageData({
-        pageData: json,
-        service,
-        variant,
-        env,
-      });
-
-      json = {
-        data: {
-          article: json,
-          // Checks for data mocked in tests, or data from fixture data
-          secondaryData: json?.secondaryData ?? {
-            topStories: secondaryData?.secondaryColumn?.topStories,
-            features: secondaryData?.secondaryColumn?.features,
-            mostRead: secondaryData?.mostRead,
-            mostWatched: secondaryData?.mostWatched,
-          },
-        },
-      };
-    }
-
-    if (!json?.data?.article) {
-      throw handleError('Article data is malformed', 500);
-    }
-
```

Comment:
I'm a bit confused, why are were we fetching data for CPS in the articles `getInitialData` anyway?

Reply:
This is probably a question for the content team. I think it's because there is a lot of shared code for articles / cpsAssets, as they both need to transform the response to the optimo blocks format? But definitely worth checking whether this is intentional. If not, we can just make the cpsAsset fetch directly from fixture data on local or BFF on test/live

Reply:
I suspect this may be to support CAF actually, may be a question for @amoore108 ? Do CAF transformed optimo articles come throught the `cpsAsset` type still @amoore108 ?

Reply:
So this was added as the logic and response structure is exactly the same between Article and CPS assets.

CPS assets don't perform any data transformations on the BFF, they all continue to happen in Simorgh. This was more to create a generic fetcher for both Article/CPS assets, which will be combined in the not too distant future with CAF. 

This logic doesn't touch CAF _yet_, but will make that transition easier.

This weird IF condition was also added so that we didn't need to update a bunch of fixture data when we quickly needed to move the CPS page fetching to the BFF awhile ago, due to traffic...issues. This is actually the main reason for this.

It probably is better to have this split out into its own `getInitialData` and keep the page types separate, but they will come together with CAF eventually.

Reply:
It appears to work for me. So I'm happy for this to be removed. It really was just something that needed added quickly at the time to get it working and maintain the local dev fixture data when we moved CPS data fetching to the BFF.

Reply:
Thanks for the responses folks. Most of the local cpsAssets will work - certainly the ones used in our unit, E2E and integration tests. I might follow up with PR by migrating other local fixtures, but I don't want to add to the 150+ files in this PR 🤦‍♀️